python/MRzeroCore/phantom/sim_data.py

```diff
@@ -150,7 +153,8 @@ def cpu(self) -> SimData:
             self.dephasing_func,
             self.recover_func,
             self.phantom_motion,
-            self.voxel_motion
+            self.voxel_motion,
+            Masks=self.Masks

```

Comment:
See cuda comment - move back to cpu here accordingly

Reply:
Done